csharp/test/Ice/metrics/ControllerI.cs

```diff
@@ -0,0 +1,29 @@
+//
+// Copyright (c) ZeroC, Inc. All rights reserved.
+//
+
+using System;
+
+namespace ZeroC.Ice.Test.Metrics
+{
+    public sealed class Controller : IController
```

Comment:
Should it? I thought our convention was files that implement Slice definitions always got at `I` at the end? Like in the same `metrics` test we also have a `MetricsI.cs` which implements the metrics interface.

Reply:
I think this make sense before because the interface was `Controller` not `IController`, but now with interface name `I<name>` I think it is fine to use `<name>` for the implementation